openedx/features/course_experience/__init__.py

```diff
@@ -33,11 +29,12 @@ def course_home_page_title(course):  # pylint: disable=unused-argument
     return _('Course')
 
 
-def default_course_url_name(request=None):
+def default_course_url_name():
     """
     Returns the default course URL name for the current user.
     """
-    if waffle.flag_is_active(request or RequestCache.get_current_request(), UNIFIED_COURSE_VIEW_FLAG):
+    # TEMPORARY NOTE: Does this break because we no longer pass through the request?
```

Comment:
Reviewers: This is just a note to self, because I am pretty sure some test is going to fail because of this.